### ISSUE TITLE
Auto Higher Bitrate selection FEATURE

### ISSUE THREAD

**Issue by BL4CKR4Y** [yuliskov/SmartTube#2970]
Hello, Can you please add automatic selection of the higher bitrate AVC 1080p and and lower, I searched in the settings but could find any option to do it.

Greetings

**Comment by MJrolla07** [comment]
I'd love to see this. I just saw in the video codecs section on the github

"At the same resolution, a lower bitrate is better! YouTube explicitly targets the same quality regardless of the codec. Older codecs have a higher bitrate only because they are less efficient. On Youtube, you do not get better quality by simply choosing a higher bitrate."

This just simply isnt true for me, I can always tell a big difference when selecting the highest bitrate. I just want an option to always automatically play the highest bitrate, highest resolution video.

**Comment by BL4CKR4Y** [comment]
Automatic higher resolution pick or playback is already working, but it lacks choosing higher bitrate, and as from my experience i noticed that the higher bitrate looks better especially for the older and low resolution contents, i don't know why u say that lower bitrate is better.

Greetings

**Comment by BL4CKR4Y** [comment]
Yeah thats what im saying, finally someone else has noticed it.

**Comment by BL4CKR4Y** [comment]
I agree with you, every time when i play 1080p and lower videos i have to manually switch to the higher bitrate, i hope a fix is added soon.
